libs/features/mas/src/variants/simplified-pricing-express.js

```diff
@@ -65,58 +70,162 @@ export class SimplifiedPricingExpress extends VariantLayout {
+            )
+        );
+
+        requestAnimationFrame(() => {
+            cards.forEach(card => card.variantLayout?.syncHeights?.());
+        });
+    }
+
+    async forceRemeasure() {
+        const container = this.getContainer();
+        if (container) {
+            await this.syncAllCardsInContainer(container);
+        }
+    }
+
+    async postCardUpdateHook() {
+        if (!this.card.isConnected) return;
+
+        await this.card.updateComplete;
+        await this.waitForPrices();
+
+        if (!!isDesktop()) {
+            const container = this.getContainer();
+            if (!container) return;
+
+            const prefix = `--consonant-merch-card-${this.card.variant}`;
+            const hasExistingVars = container.style.getPropertyValue(`${prefix}-description-height`) ||
+                                   container.style.getPropertyValue(`${prefix}-price-height`);
+
+            if (!hasExistingVars) {
+                await this.syncAllCardsInContainer(container);
+            } else {
+                this.syncHeights();
+            }
+        }
+    }
+
     connectedCallbackHook() {
         if (!this.card || this.card.failed) {
             return;
         }
-        
+
         this.setupAccordion();
-        
-        requestAnimationFrame(() => {
-            if (this.card?.hasAttribute('data-default-card') && isTabletOrBelow()) {
-                this.card.setAttribute('data-expanded', 'true');
-            }
-        });
+
+        if (this.card?.hasAttribute('data-default-card') && !isDesktop()) {
+            this.card.setAttribute('data-expanded', 'true');
+        }
     }
 
+
     setupAccordion() {
         const merchCard = this.card;
         if (!merchCard) {
             return;
         }
 
         const updateExpandedState = () => {
-            if (isTabletOrBelow()) {
+            if (!isDesktop()) {
                 const isDefaultCard = merchCard.hasAttribute('data-default-card');
                 merchCard.setAttribute('data-expanded', isDefaultCard ? 'true' : 'false');
             } else {
                 merchCard.removeAttribute('data-expanded');
             }
         };
 
-        // Set initial state
         updateExpandedState();
 
-        // Watch for viewport changes
         const mediaQuery = window.matchMedia(TABLET_DOWN);
         this.mediaQueryListener = () => {
             updateExpandedState();
         };
         mediaQuery.addEventListener('change', this.mediaQueryListener);
 
-        // Watch for default card attribute changes
         this.attributeObserver = new MutationObserver((mutations) => {
             mutations.forEach((mutation) => {
-                if (mutation.type === 'attributes' && 
-                    mutation.attributeName === 'data-default-card' && 
-                    this.card.hasAttribute('data-default-card') && 
-                    isTabletOrBelow()) {
+                if (mutation.type === 'attributes' &&
```

Comment:
I don't know why we used a mutation observer here; a reactive property/attribute dataDefaultCard/data-default-card  on merch-card should be sufficient.